### ISSUE TITLE
fromJSON not working

### ISSUE THREAD

**Issue by imcullan** [jeroen/jsonlite#173]
I am having trouble getting the JSON object using fromJSON. It was working previously but suddenly stopped working. My call is:

`shotDataJSON <- fromJSON("http://stats.nba.com/stats/shotchartdetail?PlayerID=201939&PlayerPosition=&Season=2015-16&ContextMeasure=FGA&DateFrom=&DateTo=&GameID=&GameSegment=&LastNGames=0&LeagueID=00&Location=&Month=0&OpponentTeamID=0&Outcome=&Period=0&Position=&RookieYear=&SeasonSegment=&SeasonType=Regular+Season&TeamID=0&VsConference=&VsDivision=")`
or
`allplayers <- fromJSON("http://stats.nba.com/stats/commonallplayers/?LeagueID=00&Season=2015-16&IsOnlyCurrentSeason=0")`

If anybody can help me to get it working again, that would be greatly appreciated! Thank you!

**Comment by jeroen** [comment]
This is a problem in the R http client that is triggered by the `stats.nba.com` server. It seems to have been fixed in R 3.4.

**Comment by jeroen** [comment]
To avoid the problem wrap the url in `curl::curl`:


```r
con <- curl::curl("http://stats.nba.com/stats/commonallplayers/?LeagueID=00&Season=2015-16&IsOnlyCurrentSeason=0")
allplayers <- fromJSON(con)
```
